jump/Cargo.toml

```diff
@@ -21,7 +21,7 @@ itertools = "0.10"
 log = { workspace = true }
 logging_timer = { workspace = true }
 memmap2 = "0.7"
-os_str_bytes = "6.5"
+os_str_bytes = { version = "6.6", features = ["conversions"] }
```

Comment:
This was prompted by a deprecation warning and otherwise unrelated to this change.